package.json

```diff
@@ -84,6 +84,7 @@
     "react-router-dom": "^4.1.2",
     "react-tag-autocomplete": "^5.4.1",
     "redux": "^3.7.2",
+    "redux-devtools-extension": "^2.13.2",
     "redux-thunk": "^2.2.0",
```

Comment:
@saubhagya23 : this tool is only used for Development purpose. So add it in dev dependencies rather than dependencies.